tests/lib.rs

```diff
@@ -67,7 +68,10 @@ fn httpbin_transaction<D: Deliverable>(deliverable: D) -> Transaction<D> {
 
 fn check_successful_result(result: DeliveryResult) -> (bool, DeliveryResult) {
     let successful = match result {
-        DeliveryResult::Response { ref response, .. } => response.status().is_success(),
+        DeliveryResult::Response { ref response, .. } => {
+            // Onesignal is returning 302
+            response.status().is_success() || response.status().is_redirection()
```

Comment:
Is it expected to get a redirect?
It might be better to either change the request if it's outdated, or if it's expected we should assert the redirect `location` to make sure it's a correct one

Reply:
I tried `https://onesignal.com/` and it seems to work, so I don't see a redirect in the browser.